src/red.ts

```diff
@@ -329,7 +326,7 @@ export const serializedRedEnvSourceText = (function redEnvFactory(blueEnv: Membr
                 try {
                     // the error constructor must be a blue error since it occur when calling
                     // a function from the blue realm.
-                    const redErrorConstructor = blueEnv.getRedRef(constructor);
+                    const redErrorConstructor = blueBroker.getRedRef(constructor);
                     // the red constructor must be registered (done during construction of env)
                     // otherwise we need to fallback to a regular error.
                     redError = construct(redErrorConstructor as RedFunction, [message]);
```

Comment:
☝️ what color is that `construct`? blue?

Reply:
it is not imported, it is destructed above from the red reflect, so it is red. we could rename them all, but... it is from the closure

Reply:
Ok, Ya a clear color code for it may be helpful.